packages/react/src/wishlists/hooks/__tests__/useWishlistItem.test.tsx

```diff
@@ -40,11 +40,14 @@ describe('useWishlistItem', () => {
     const current = getRenderedHook();
 
     expect(current).toStrictEqual({
-      removeWishlistItem: expect.any(Function),
-      updateWishlistItem: expect.any(Function),
       error: expect.any(Object),
       isLoading: expect.any(Boolean),
-      item: expect.any(Object),
+      isFetched: expect.any(Boolean),
+      actions: {
+        update: expect.any(Function),
+        remove: expect.any(Function),
+      },
+      data: expect.any(Object),
```

Comment:
Same as with the `useWishlist` hook, instead of expecting any object to be in data, what do you think of checking for the exact value from the mock state that you expect?